### PR TITLE
Fix mob tool warnings

### PR DESCRIPTION
**Pull request by kylefleming**
### Motivation

This PR addresses a number of Python and PEP8 warnings in the `mob` tool.

### In this PR
* Fix Python warnings
* Fix PEP8 warnings
  * E127: Continuation line over-indented for visual indent
  * E251: Unexpected spaces around keyword / parameter equals
  * E261: At least two spaces before inline comment
* Remove unused `bash_commands` variable
* Various other minor cleanup

### Future Work
* This PR does not address all PEP issues

### CODE REVIEW

File: mob [hunk 1]
```diff
@@ -240,23 +241,10 @@ if minor_ver > 0:
...
-            "RUST_BACKTRACE=full",
-            "SGX_MODE=HW" if args.hw else "SGX_MODE=SW",
-            "IAS_MODE=PROD" if args.ias_prod else "IAS_MODE=DEV",
-            ]
-
-# Calculate bash commands
-if args.action == "prompt" or args.action == "image":
-    bash_commands = None
-else:
+if not (args.action == "prompt" or args.action == "image"):
```
Comment (eranrund) [inline mob:247]: `if args.actions not in ['prompt', 'mage']:`

File: mob [hunk 2]
```diff
@@ -183,7 +183,7 @@ if not args.dry_run:
         if retcode != 0:
             if not args.verbose:
                 eprint(cmd)
-            eprint("docker not found: retcode =", retcode)
+            eprint("docker not found: retcode = ", retcode)
```
Comment (eranrund) [inline mob:186]: I believe this will print with an extra space, for example:
```
>>> print('a', 10)
a 10
```

File: mob [hunk 3]
```diff
@@ -219,18 +219,19 @@ if 'dockerfile' in mobconf['image']:
...
+    docker_dir = None
     version_data = mobconf['image']['Dockerfile-version']
 else:
     raise Exception("Neither 'dockerfile' nor 'Dockerfile-version' was found in .mobconf")
 
 # Parse what version data we found
 ver_segments = version_data.split('_')
 if len(ver_segments) != 2:
-    raise Exception("Expected form A_B for Dockerfile-version, found '" + data + "' which parsed as: " + str(ver_segments))
+    raise Exception("Expected form A_B for Dockerfile-version, found '" + version_data + "' which parsed as: " + str(ver_segments))
```
Comment (eranrund) [inline mob:230]: This should use string formatting. Either 'bleh bleh {} bleh'.format(version_data, ver_segments) or if only supporting Python>3.7, `f'bleh bleh {version_data}'`

File: mob (hunk 1 above)
Comment (kylefleming) [reply mob]: After investigating more, I believe this condition is unreachable anyway, so I removed it.

File: mob (hunk 3 above)
Comment (kylefleming) [reply mob:230]: The rest of the file tends to use string concatenation, so I think if we want to improve it, we should do a pass at the whole file to keep it consistent.

File: mob (hunk 1 above)
Comment (kylefleming) [reply mob]: (because of argparse: `parser.add_argument("action", choices=["prompt", "image", "default-tag"], ...`)

**Pull request merged**